src/PokeItem.js

```diff
@@ -4,11 +4,12 @@ class PokeItem extends Component {
     
     render() { 
         //whole object
-        const{url, pokemon, _id} = this.props;
+        const{url, pokemon, _id, ability} = this.props;
         return ( 
 
             <p>
```

Comment:
Here instead of the p tag (or in addition to it if you want) you'd want to put the Link so each pokemon in the list becomes a link to the details page